index.html

```diff
@@ -1866,12 +1851,15 @@ <h3> 'Subject Only' Terms of Use </h3>
       </p>
 
   <p>
-The Subject Only Terms of Use states that a verifiable credential MUST only be
-presented to a verifier by the subject. If a verifier is presented with a
-verifiable credential containing the Subject Only Terms of Use, by anyone other
-    than the subject, it MUST refuse to accept it.
+The Subject Only Terms of Use states that a verifiable credential is only to be presented by the subject. If a verifier is presented with a verifiable presentation containing a verifiable credential with a Subject Only Terms of Use, by anyone other
```

Comment:
This particular case seems similar to the verification rules (section 8) related to revocation where the DM states in section 8.8, Revocation: "If revocation instructions are present, the credential must not have been revoked."  In this case, "If the ToU type is Subject only, the credential must have been presented by the Subject".  The implication here is that if the opposite is true, the it should fail to "verify" -- the idea and language of "acceptable" is a Fitness for Purpose argument and it's appropriate.

Reply:
Hmmmm...

What is being verified by evaluating the ToU? 

The semantics of VC verification, IMO, mean verifying that the utterance is authentic (crypto check) and still valid (revocation check).

Evaluating the ToU is something done once you know the underlying credential is authentic. After all, if it isn't authentic, it may not be the actual current terms from the alleged issuer. 

Seems to me there is a big difference between "Is this statement authentic and current?" and "Can we use this statement legally for our purposes?"

The latter is about the use of the VC, not its authenticity.

In short, isn't verification about authenticity?

Reply:
@jandrieu Authentic and current is different from simply authentic. An expired VC is still authentic. So if you want to add "current" to the authenticity test, then you can also add "presented by the subject" to the authenticity test. This is nothing to do with legality. It is about testing properties of a VC, such as: not revoked, not expired, not presented by the subject, and anything else of generic use and value that we may wish to standardise.